### PR TITLE
feat(api): Note CRUD

### PR DESCRIPTION
**Pull request by lambdina**
# Description

Notes CRUD. Issue number 

# Changes 
- [ ] New feature : Notes CRUD
- [ ] Breaking change : Rename Customer into Pentester

# Breaking changes

Rename Customer into Pentester.

# Checklist

- [ x ] I have assigned this PR to myself
- [ x ] I have added @vrn-sh/members to the reviewers, or specific members of the team
- [ x ] I have added the needed labels
- [ x ] I have tested this code
- [ x ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)

### CODE REVIEW

File: api/api/migrations/0001_initial.py [hunk 1]
```diff
@@ -49,18 +49,28 @@ class Migration(migrations.Migration):
             ],
         ),
         migrations.CreateModel(
-            name='Customer',
+            name='Pentester',
```
Comment (djnnvx) [inline api/api/migrations/0001_initial.py:52]: why not create a new migration instead? bc of this i will have to delete prod database xD

File: api/api/models/vulns.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+MAX_NOTE_LENGTH = 8186
+
+
+class Notes(models.Model):
+    """
+        In order to give pentesters flexibility in their work, we should let them save simple notes about the
+        current infrastructure.
+    """
+
```
Comment (djnnvx) [inline api/api/models/vulns.py:13]: please add `REQUIRED_FIELDS` so Swagger documentation stays up to date

**Review by djnnvx — CHANGES_REQUESTED** [review]
fix the minor details & fill out the pull request template for automations please

then OK to merge :ok_hand:

File: api/api/migrations/0001_initial.py (hunk 1 above)
Comment (lambdina) [reply api/api/migrations/0001_initial.py:52]: J'ai commit d'un coup donc si je checkout je vais perdre mes autres modifs. Comment je peux faire?

File: api/api/models/vulns.py (hunk 2 above)
Comment (lambdina) [reply api/api/models/vulns.py:13]: done

**Review by djnnvx — APPROVED** [review]

**Pull request merged**